### ISSUE TITLE
snapshots components

### ISSUE THREAD

**Issue by talpa** [Memnarch/Delphinus#61]
It is posible make components snapshots for current project like php composer?

**Comment by talpa** [comment]
yes via git commit hash. for example information save like json  "source": {
                "type": "git",
                "url": "gitolite@trac.geesoft.net:geesoftnet/database",
                "reference": "f25617fd59fca3c5f367d6956da56ec715526cf3"
            }

**Comment by Memnarch** [comment]
No that's not possible. In the future i might think about adding project-configurations. But referencing by commit is not supposed to work at any time. If i add Projectconfigurations(so that you can use different VirtualTreeViews in different projects) it'll work by linking with the version of the package.
